webapp-django/crashstats/crashstats/static/crashstats/js/socorro/correlation.js

```diff
@@ -197,7 +197,7 @@ window.correlations = (function() {
             channel +
             '" channel, for the "' +
             product +
-            '" product.'
+            '" product. The number of crashes was probably too low in this channel. Please try to select another channel.'
```

Comment:
If the signature is long, the text overflows outside the container:

<img width="1625" alt="screen shot 2018-06-06 at 2 30 03 pm" src="https://user-images.githubusercontent.com/193106/41066333-5c380f9a-6996-11e8-8f33-b63329eb4e4c.png">

This isn't a problem with this PR per se, it's just made more apparently now that the text is longer.

The root fix for this would be to fix the CSS/HTML around that tag and allow it to wrap. I dunno if we don't want it to be a `<pre>` tag though, as that might break the normal correlation output. An acceptable fix for the purposes of this PR would be to add a newline or two in front of the new string in order to break it out into its own line.

Reply:
Thanks! I will fix it.